grails-app/services/au/org/ala/phyloviz/TreeService.groovy

```diff
@@ -628,4 +628,8 @@ class TreeService {
         }
         return result;
     }
+
+    def deleteTree(Integer treeId) {
```

Comment:
looks like anyone can delete this tree. But I think only the creator or admin should be able to do it.